Screen_OLED_Test/Screen_OLED_Test.ino

```diff
@@ -0,0 +1,19 @@
+#include "OLEDScreen.h"
+
+//7.5 hours; reading library documentation and understanding
```

Comment:
Please add this in your gantt chart. And please add your global pinLayout and control variables like the State(followerMode, autonomusMode, manualMode; best used as enum).